contracts/optimize/OptimizedLargeStorageManager.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./SlotHelper.sol";
+import "../StorageHelper.sol";
+import "../StorageSlotSelfDestructable.sol";
+
+contract OptimizedLargeStorageManager{
```

Comment:
I feel like copying the code from LargeStorageManager is not a good practice.  Could we inherit LargeStorageManager and override the methods?

Reply:
Is it necessary to inherit when we need to override all methods?

Reply:
yes ,we could ! Inheritance is a better idea than just copying code!

Reply:
https://github.com/QuarkChain/evm-large-storage/pull/1/commits/58f21bccabd3ad1dc161a779d5c9a2889502217c

Reply:
https://github.com/QuarkChain/evm-large-storage/pull/1/commits/d4f11d97eb07298ecd9bad59153ef865f74564fa